fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -572,7 +572,7 @@ private synchronized void receiveInteraction(VehicleSlowDown vehicleSlowDown) th
             );
         }
         bridge.getVehicleControl()
-                .slowDown(vehicleSlowDown.getVehicleId(), vehicleSlowDown.getSpeed(), (int) vehicleSlowDown.getInterval());
+                .slowDown(vehicleSlowDown.getVehicleId(), vehicleSlowDown.getSpeed(), convertTime((int) vehicleSlowDown.getInterval()));
```

Comment:
`vehicleSlowDown.getInterval()` returns the interval as seconds in `double`. casting this to int would break precision (e.g. 
`0.1` seconds would be cast to `0`). This also might be the reason why Barnim integration test fails on Jenkins.